graphql/util.go

```diff
@@ -543,6 +544,35 @@ func generateBuildVariants(sc data.Connector, versionId string, searchVariants [
 	return result, nil
 }
 
+// getFailedTestResultsSample returns a sample of failed test results for the given tasks that match the supplied testFilters
+func getCedarFailedTestResultsSample(ctx context.Context, tasks []task.Task, testFilters []string) ([]apimodels.CedarFailedTestResultsSample, error) {
+	if len(tasks) == 0 {
+		return nil, nil
+	}
+	taskFilters := []apimodels.CedarTaskInfo{}
+	for _, t := range tasks {
+		taskFilter := apimodels.CedarTaskInfo{
+			TaskID:      t.Id,
+			Execution:   t.Execution,
+			DisplayTask: t.DisplayOnly,
+		}
+		taskFilters = append(taskFilters, taskFilter)
```

Comment:
Nit: since you are not reusing `taskFilter`, you can combine this with the lines above.
```
taskFilters = append(taskFilters, apimodels.CedarTaskInfo{
        TaskID: t.Id,
        Execution: t.Execution,
        DisplayTask: t.DisplayOnly,
})
```